tests/unit/fragment_array_test.js

```diff
@@ -282,4 +282,66 @@ module('unit - `MF.fragmentArray`', function(hooks) {
       });
     });
   });
+
+  test('can be created with null', function(assert) {
+    let person = store.push({
+      data: {
+        type: 'person',
+        id: 1,
+        attributes: {
+          names: null
+        }
+      }
+    });
+
+    run(() => {
+      assert.strictEqual(person.names, null);
+    });
+  });
+
+  test('can be updated to null', function(assert) {
+    let person = store.push({
+      data: {
+        type: 'person',
+        id: 1,
+        attributes: {
+          names: [
+            {
+              first: 'Catelyn',
+              last: 'Tully'
+            },
+            {
+              first: 'Catelyn',
+              last: 'Stark'
+            }
+          ]
+        }
+      }
+    });
+
+    run(() => {
+      assert.deepEqual(person.names.toArray().map((f) => f.serialize()), [
+        {
+          'first': 'Catelyn',
+          'last': 'Tully'
+        },
+        {
+          'first': 'Catelyn',
+          'last': 'Stark'
+        }
+      ]);
+
+      store.push({
+        data: {
+          type: 'person',
+          id: 1,
+          attributes: {
+            names: null
+          }
+        }
+      });
+
+      assert.strictEqual(person.names, null);
```

Comment:
so the current FragmentArray behavior in the master is that `person.names` are equal to an empty fragment array wrapper:
```
      assert.deepEqual(person.names.toArray().map((f) => f.serialize()), []);
```
So the change from this PR is technically a breaking change. 

However, IMO this is not consistent with how it behaves for the creation case, where the `names` are equal to `null`(https://github.com/adopted-ember-addons/ember-data-model-fragments/pull/415/files#diff-ab7b846c8b4a5b9482d558f599ee7fe427d8f12d6b9f3b7f00b506cd78d76bf6R298). I tend to leave this change for the sake of consistency. But, please, let me know if should preserve the existing behavior.

Reply:
Thanks @ro0gr, yep I agree that consistency is better and fragmentArrays can be nullable both when they are created and when they are updated. Technically we are still in the 5.x beta phase so a breaking change should be fine.

@knownasilya do you have thoughts on this slight change in behaviour?

Reply:
Sounds fine to me, as long as we document the change.

Reply:
@knownasilya ok, I looked through the README and could not find a proper place where this detail could be injected into(in a natural way). I would gladly update the README(or whatever other documentation source) if there is a good idea on how to do it.

Meanwhile, I've updated the commit message, and PR description to contain a `[breaking]` label, + provided some more context regarding the reasons for this change. So I expect it to become a part of the changelog.
Also at this point we've fixed the intended behavior in tests, which could be approached as a kind of documentation as well. I think a `[breaking]` entry in the changelog + tests may be a good enough source of documentation. WDYT?

Reply:
Documenting it as `[breaking]` in the changelog sounds good enough to me 👍